core/gui/src/app/workspace/service/workflow-result-export/workflow-result-export.service.ts

```diff
@@ -145,56 +146,61 @@ export class WorkflowResultExportService {
       return;
     }
 
-    let operatorIds: string[];
-    if (!exportAll)
+    // gather operator IDs
+    let operatorIds: string[] = [];
```

Comment:
try to rewrite this using some syntax sugar to let this `operatorIds` be a `const operatorIds`